scripts/build-image.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+PREFER_PODMAN="${PREFER_PODMAN:-0}"
```

Comment:
This should be enabled by default

Reply:
Will update `PREFER_PODMAN="${PREFER_PODMAN:-1}"` in the next commit to make it used by default.